scripts/ci/jobs/jobs.yml

```diff
@@ -487,6 +492,8 @@
     builders:
       - skydive-test:
           test: scripts/ci/create-docker-image.sh
+      - skydive-test:
+          test: WITH_SHA1=true scripts/ci/create-docker-image.sh
```

Comment:
The images will have different ids. Why don't we simply run the script once and push 2 images ?

Reply:
having the script just push both to skydive and skydive-snapshots has some cons:

(1) test runs take twice as long
(2) code of create-docker-image more complex
(3) a duplication of input environment parameters:

```
export DOCKER_IMAGE=abc/skydive
export DOCKER_USERNAME=abc
export DOCKER_PASSWORD=...

export DOCKER_IMAGE_SNAP=xyz/skydive-snapshots
export DOCKER_USERNAME_SNAP=xyz
export DOCKER_PASSWORD_SNAP=...
```

so have the job call the same script twice is much more elegant and more maintainable.

Reply:
But having the 2 images have 2 different ID is a bigger disadvantage to me (more space is used). Besides, I really don't think it makes the script more complexe.
Why would duplicate DOCKER_USERNAME and DOCKER_PASSWORD ?

Reply:
ok - have fixed